src/autoupdater.ts

```diff
@@ -55,6 +60,34 @@ export class AutoUpdater {
     return isUpdated;
   }
 
+  async handleSchedule(): Promise<number> {
+    const ref = this.config.getValue('GITHUB_REF');
+    const ownerAndRepo = this.config.getValue('GITHUB_REPOSITORY');
+
+    if (ownerAndRepo === undefined) {
+      ghCore.error('GITHUB_REPOSITORY value is undefined.');
+      return 0;
+    }
+
+    if (ref === undefined) {
+      ghCore.error('GITHUB_REF value is undefined.');
+      return 0;
+    }
+
+    const splitRepoName = ownerAndRepo.split('/');
+    const repoOwner = splitRepoName[0];
+    const repoName = splitRepoName[1];
+
+    if (repoOwner === undefined || repoName === undefined) {
```

Comment:
`repoOwner` will never be undefined, maybe a length check is better here? i.e. 
```suggestion
    if (splitRepoName.length !== 2) {
```